sycl-jit/jit-compiler/lib/rtc/DeviceCompilation.cpp

```diff
@@ -831,30 +831,38 @@ Error jit_compiler::linkDeviceLibraries(llvm::Module &Module,
   DiagnosticsEngine Diags(DiagID, DiagOpts, Wrapper.consumer(),
                           /* ShouldOwnClient=*/false);
 
+  Triple T{Module.getTargetTriple()};
+  Driver D{(SYCLToolchain::instance().getPrefix() + "/bin/clang++").str(),
+           T.getTriple(), Diags};
```

Comment:
There is `getClangXXExe`: https://github.com/intel/llvm/blob/e711ac0bfed7e8168110399c16b48cc1221b8626/sycl-jit/jit-compiler/lib/rtc/DeviceCompilation.cpp#L616-L617

I understand this is a move, but we can update it still.